### DISCUSSION TITLE
Stenograpther is listen to bond interface that seems to have no members after install

### DISCUSSION THREAD

**Discussion by dsbrodbeck** [Security-Onion-Solutions/securityonion#3324] [Unsupported Versions] [answered]
Hello,
     I installed SO 2.3 on a Ubuntu 18.04 installation with one management and 3 listening interfaces on a Standalone installation.  I see that Stenographer is configured to listen to the bond0 interface.  I check the netplan and I don't see the bond configured. tcpdumping on bond0 indicates no traffic.  Tcpdumping on the listening interfaces shows the expected traffic.  Am I missing something?  I tried running the so-setup-network a second time with no effect.  Steno is creating 0 byte files.

Thanks,
David

**Comment by TOoSmOotH** [comment, marked as answer]
This is because of netplan and network manager fighting over the monitor interfaces. Remove the interfaces from netplan and restart networking. Then use nmcli to add the monitor interfaces to bond0.

**Reply by dsbrodbeck** [reply]
Thank You, this was the problem.  I commented out the interfaces in the Netplan config file, and then added the interfaces via nmcli.  And bond0 started receiving all of the traffic expected.

-David
